src/errors/error-messages.ts

```diff
@@ -72,3 +75,16 @@ export const counterPartyPoolErrorMessage = {
   ) =>
     `Pool ${poolId.toString()} status is ${poolStatus}, it must be ${expectedStatus.sort().join(', or ')}`
 }
+
+export const stakingErrorMessage = {
+  nonNegativeLockDuration: 'Lock duration cannot be negative',
+  nonNegativeApy: 'APY cannot be negative',
+  invalidPlan: 'Invalid plan',
```

Comment:
Maybe it could be clearer if we specify that it is an index, `Invalid plan index`